sinatra-active-model-serializers.gemspec

```diff
@@ -13,33 +13,43 @@ Gem::Specification.new do |s|
   s.license = 'MIT'
 
   # generated from git shortlog -sn
-  s.authors = ['saulosantiago']
+  s.authors = ['Saulo Santiago', 'Adriano Tadao']
```

Comment:
@SauloSilva  Ohh man! In this way you'll make me cry! =D This gem is your son! Is a result of your big dedication, but, thanks for your consideration. =D